modules/hivemq-edge-module-opcua/src/main/java/com/hivemq/edge/adapters/opcua/client/OpcUaEndpointFilter.java

```diff
@@ -71,9 +71,16 @@ public OpcUaEndpointFilter(@NotNull String configPolicyUri, @NotNull OpcUaAdapte
 
     private EndpointDescription endpointUpdater(EndpointDescription endpoint) {
         if (adapterConfig.getOverrideUri()) {
-            return EndpointUtil.updateUrl(endpoint,
+
+            final EndpointDescription endpointDescription = EndpointUtil.updateUrl(endpoint,
                     EndpointUtil.getHost(adapterConfig.getUri()),
                     EndpointUtil.getPort(adapterConfig.getUri()));
+
+            if(log.isInfoEnabled()) {
+                log.info("Overwriting returned endpoint {} with {}", endpoint, endpointDescription);
+            }
+
+            return endpointDescription;
         }
         return endpoint;
     }
```

Comment:
Formatting

Reply:
What exactly?

Reply:
i think he means the missing space after the 'if'

Reply:
Thanks, fixed.